Blish HUD/GameServices/Contexts/Gw2ClientContext.cs

```diff
@@ -0,0 +1,92 @@
+﻿namespace Blish_HUD.Contexts {
+
+    public class Gw2ClientContext : Context {
+
+        /// <summary>
+        /// The type of the client currently running.
+        /// </summary>
+        public enum ClientType {
+            /// <summary>
+            /// The client type could not be determined.
+            /// </summary>
+            Unknown,
+
+            /// <summary>
+            /// The client is the standard US/EU varient of the client.
+            /// </summary>
+            Standard,
+
+            /// <summary>
+            /// The client is the Chinese varient of the client.
+            /// </summary>
+            Chinese
+        }
+
+        /// <inheritdoc />
+        protected override void Load() {
+            this.ConfirmReady();
+        }
+
+        #region Specific Checks
+
+        private bool IsStandardClientType(int currentBuildId, out ContextAvailability contextAvailability) {
+            contextAvailability = GameService.Contexts.GetContext<CdnInfoContext>().TryGetStandardCdnInfo(out var standardCdnContextResult);
+
+            if (contextAvailability == ContextAvailability.Available) {
+                int standardBuildId = standardCdnContextResult.Value.BuildId;
+
+                if (currentBuildId == standardBuildId) {
+                    return true;
+                }
+            }
+
+            return false;
```

Comment:
This is probably preference, I would write that a little bit different for readability:

```cs
if (contextAvailability != ContextAvailability.Available)
    return false;

var standardBuildId = standardCdnContextResult.Value.BuildId;
return currentBuildId == standardBuildId;
```

Reply:
I didnt want to change too much, but yes, I would omit the var as well